### ISSUE TITLE
Subtitle font size

### ISSUE THREAD

**Issue by philipparkergit** [mycoralhealth/mycoral-patient#15]
Can you make the subtitles a larger font? Hard to read them.

**Comment by grcevski** [comment]
Hey @philipparkergit I tried, but the text on the delegate access screen won't fit. We either have to keep it as is or reword that message to be shorter.

**Comment by philipparkergit** [comment]
Hey @grcevski gotcha. How about we reword it to say "Scan the recipient’s QR code or enter their address to share this record."? Is that short enough?

Philip

**Comment by lukecyca** [comment]
My intention with the subtitle was as a one-liner. 4 words max.
The text is not large enough for critical instructions, and there is simply not enough room anyway.

Let's make them all way shorter, and move critical instructions down to the body of the page.

**Comment by nosequeldeebee** [comment]
Luke will move instructions down to the page and have the title be succinct

**Comment by grcevski** [comment]
Sounds good, I removed myself as owner, feel free to grab guys :)!
